server/src/handlers.py

```diff
@@ -14,26 +14,30 @@ async def ping(request: web.Request) -> web.Response:
 
 async def create_room(request: web.Request) -> web.Response:
     data = await request.json()
-    some_room = room.Room(data)
+    room = room_model.Room(data)
     try:
-        some_room.validate()
+        room.validate()
     except DataError as e:
         return web.json_response(e.to_primitive(), status=400)
-    if await db.room_is_occupied(data):
+    pool = request.app['pool']
+    room_exist = await db.is_room_exist(pool, data)
+    if room_exist:
         return web.json_response({'msg': 'room with that name already exists'}, status=409)
-    await db.create_room(data)
-    return web.json_response({'msg': 'OK'}, status=201)
+    room_id = await db.create_room(pool, data)
+    return web.json_response({'id': room_id}, status=201, dumps=parse_json)
```

Comment:
We h've use parse_json to dump ObjectID, for now we don't have that problem, so it can be removed